CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+ - [#1159](https://github.com/Shopify/shopify-api-ruby/pull/1159) Backport fixes around computing attribute changesets when saving REST resources
```

Comment:
I assume this changelog is later modified to include the version the changes are captured in?